lib/bolt/pal/yaml_plan/transpiler.rb

```diff
@@ -21,10 +21,18 @@ def transpile(relative_path)
           validate_path
 
           plan_object = parse_plan
+          param_descriptions = plan_object.parameters.each_with_object([]) do |param, acc|
```

Comment:
This can just be `map` rather than `each_with_object` with an accumulator, since the results are being joined at the end anyhow.

Reply:
Updated